flang/test/Semantics/OpenMP/omp_taskgroup_allocate.f90

```diff
@@ -0,0 +1,12 @@
+! RUN: %flang_fc1 -fopenmp -fsyntax-only %s
```

Comment:
Can you run with test_errors.py to ensure that no Error messages are generated?

Reply:
Sure. I have tested it with test_error.py and no error are generated.
I haven't used test_error.py for validating this testcase by referring the below comment https://github.com/llvm/llvm-project/pull/109265/files#r1767132309.